<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>DeviceManagerDialog</class>
    <widget class="QDialog" name="DeviceManagerDialog">
        <property name="windowTitle">
            <string>Secrum â€” Devices</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="titleLabel">
                    <property name="text">
                        <string>Active devices</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                    <property name="margin">
                        <number>4</number>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QTableWidget" name="devicesTable">
                    <property name="selectionBehavior">
                        <enum>QAbstractItemView::SelectRows</enum>
                    </property>
                    <property name="selectionMode">
                        <enum>QAbstractItemView::SingleSelection</enum>
                    </property>
                    <property name="editTriggers">
                        <set>QAbstractItemView::NoEditTriggers</set>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="buttonLayout">
                    <item>
                        <widget class="QPushButton" name="refreshButton">
                            <property name="text">
                                <string>Refresh</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="deactivateButton">
                            <property name="text">
                                <string>Deactivate</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="deleteButton">
                            <property name="text">
                                <string>Delete</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="closeButton">
                            <property name="text">
                                <string>Close</string>
                            </property>
                            <property name="autoDefault">
                                <bool>false</bool>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <connections>
        <connection>
            <sender>closeButton</sender>
            <signal>clicked()</signal>
            <receiver>DeviceManagerDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>10</x>
                    <y>10</y>
                </hint>
                <hint type="destinationlabel">
                    <x>10</x>
                    <y>10</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>